controllers/validationController.js

```diff
@@ -41,14 +41,33 @@ exports.validateAdminPassword = (req, res, next) => {
     // console.log("adminPassword", adminPassword);
     const password = req.query.password
     // console.log("password", password);
-    if (password === adminPassword) {
-        // then good to go to players  
-        next();
-    } else {
+    
+    // get headers authorization
+    const authheader = req.headers.authorization;
+    console.log('req.headers.authorization', authheader);
+    // and distract only token password
+    const tokenPass = authheader.split(' ')[1]
```

Comment:
```
authheader.split(' ')[1]
```
This will crash when `authheader` is `undefined`, won't it? Considering there is still an old web version that sends the password via query string. In this case, the server will crash. We should null check on this.